application/migrations/env.py

```
@@ -0,0 +1,108 @@
+from importlib import import_module
```

Ruff (0.8.2)

Missing docstring in public module

(D100)